maven-core/src/main/java/org/apache/maven/graph/DefaultGraphBuilder.java

```diff
@@ -254,12 +215,13 @@ private List<MavenProject> trimResumedProjects( List<MavenProject> projects, Pro
 
         if ( StringUtils.isNotEmpty( request.getResumeFrom() ) )
         {
-            File reactorDirectory = getReactorDirectory( request );
+            File reactorDirectory = !isNullOrEmpty( request.getBaseDirectory() )
```

Comment:
Certain? `new File( "" )` isn't wrong, no?

Reply:
Good catch, you are right. It was intended to be the follow the [original implementation](https://github.com/apache/maven/blob/master/maven-core/src/main/java/org/apache/maven/graph/DefaultGraphBuilder.java#L436) but slipped through.